Test/ContentPipeline/BitmapContentTests.cs

```diff
@@ -174,6 +174,9 @@ public void BitmapConvertFullNoResize()
         }
 
         [Test]
+#if DESKTOPGL
+        [Ignore("NVidia native library could not be loaded on Mac build server.")]
```

Comment:
@tomspilman what is the problem here? I had these working before.

Reply:
Don't know... exactly like i said it wasn't loading the library on the Mac build server.

However we have bigger issues to solve first as i cannot run the latest NUnit on the Mac keeping us from running 95% of our tests.

Reply:
These should work after a rebase/merge because we switched to the managed implementation in #6102.

Reply:
@Jjagg - But we're switching back to NVTT soon.  https://github.com/MonoGame/MonoGame/pull/6102#issuecomment-368181625